packages/elements/src/components/ui/action-menu/ActionMenuItemContent.tsx

```diff
@@ -62,7 +62,6 @@ export const ActionMenuItemContent = forwardRef<
           </Row>
         )}
       </Column>
-      <Spacing />
```

Comment:
Keep this, I think. The gap differs between different types of action menu items, so the this is in addition to the gap=1 in action menu.

Reply:
Are we doing something else wrong here then? 
![image](https://github.com/StenaIT/stenajs-webui/assets/817046/a326e2a5-6dbf-4be5-90d0-cfb3c197adac)

Reply:
ouff